improver_tests/nbhood/square_kernel/test_SquareNeighbourhood.py

```diff
@@ -52,12 +52,12 @@ def test_sum_or_fraction(self):
         sum_or_fraction = "nonsense"
         msg = "option is invalid"
         with self.assertRaisesRegex(ValueError, msg):
-            Neighbourhood("square", sum_or_fraction=sum_or_fraction)
+            NeighbourhoodProcessing("square", 2000, sum_or_fraction=sum_or_fraction)
 
 
-class Test_run(IrisTest):
+class Test_process(IrisTest):
 
-    """Test the run method on the Neighbourhood class."""
+    """Test the process method on the NeighbourhoodProcessing class."""
```

Comment:
I've re-labelled the run method to process to make it work better with the simplified BaseNeighbourhood class